files/botautoresponse.json

```diff
@@ -762,4 +762,4 @@
         ],
         "response": "https://maven.minecraftforge.net/net/minecraftforge/forge/1.8.9-11.15.1.2318-1.8.9/forge-1.8.9-11.15.1.2318-1.8.9-installer.jar"
     }
-]
+]
```

Comment:
:suspect:

Reply:
idk what happened there it says that i edited lines that i didnt edit

Reply:
![image](https://user-images.githubusercontent.com/10727862/178086683-0b05ef9f-3458-495e-bc29-a6f4e88cefcc.png)

Reply:
my bad it does that when i save the file for some reason, prettier probably

Reply:
Prettier adds lines at the end of files IIRC, check your global config

Reply:
you were right, prettier was disabled for some reason idek anymore my brain is going to explode